rmgpy/statmech/conformerTest.py

```diff
@@ -36,75 +36,80 @@
 import numpy
 
 from rmgpy.statmech import Conformer, IdealGasTranslation, NonlinearRotor, HarmonicOscillator, \
-                           LinearRotor, HinderedRotor 
+    LinearRotor, HinderedRotor
 import rmgpy.constants as constants
 
 ################################################################################
 
+
 class TestConformer(unittest.TestCase):
     """
     Contains unit tests of the :class:`Conformer` class.
     """
-    
+
     def setUp(self):
         """
         A function run before each unit test in this class.
         """
         self.ethylene = Conformer(
-            E0 = (0.0,"kJ/mol"),
-            modes = [
-                IdealGasTranslation(mass=(28.03,"amu")),
-                NonlinearRotor(inertia=([3.41526,16.6498,20.065],"amu*angstrom^2"), symmetry=4),
-                HarmonicOscillator(frequencies=([828.397,970.652,977.223,1052.93,1233.55,1367.56,1465.09,1672.25,3098.46,3111.7,3165.79,3193.54],"cm^-1")),
+            E0=(0.0, "kJ/mol"),
+            modes=[
+                IdealGasTranslation(mass=(28.03, "amu")),
+                NonlinearRotor(inertia=([3.41526, 16.6498, 20.065], "amu*angstrom^2"), symmetry=4),
+                HarmonicOscillator(frequencies=([828.397, 970.652, 977.223, 1052.93, 1233.55,
+                                                 1367.56, 1465.09, 1672.25, 3098.46, 3111.7, 3165.79, 3193.54], "cm^-1")),
             ],
-            spinMultiplicity = 1,
-            opticalIsomers = 1,
+            spinMultiplicity=1,
+            opticalIsomers=1,
         )
         self.oxygen = Conformer(
-            E0 = (0.0,"kJ/mol"),
-            modes = [
-                IdealGasTranslation(mass=(31.99,"amu")),
-                LinearRotor(inertia=(11.6056,"amu*angstrom^2"), symmetry=2),
-                HarmonicOscillator(frequencies=([1621.54],"cm^-1")),
+            E0=(0.0, "kJ/mol"),
+            modes=[
+                IdealGasTranslation(mass=(31.99, "amu")),
+                LinearRotor(inertia=(11.6056, "amu*angstrom^2"), symmetry=2),
+                HarmonicOscillator(frequencies=([1621.54], "cm^-1")),
             ],
-            spinMultiplicity = 3,
-            opticalIsomers = 1,
+            spinMultiplicity=3,
+            opticalIsomers=1,
         )
-        
+
         # The following data is for ethane at the CBS-QB3 level
         self.coordinates = numpy.array([
-            [  0.0000,  0.0000,  0.0000],
-            [ -0.0000, -0.0000,  1.0936],
-            [  1.0430, -0.0000, -0.3288],
-            [ -0.4484,  0.9417, -0.3288],
-            [ -0.7609, -1.2051, -0.5580],
-            [ -0.7609, -1.2051, -1.6516],
-            [ -0.3125, -2.1468, -0.2292],
-            [ -1.8039, -1.2051, -0.2293],
+            [0.0000,  0.0000,  0.0000],
```

Comment:
I think the original intention was to have all numbers align nicely in these lists. Since we're not doing that anymore with the repetitive spaces, could you replace any multiple spaces `_ _` with a single one `_`?
(I'm using the `_` sign for a space for visibility in this comment)

Reply:
Thank Alon for changing it.